config/config.txt

```diff
@@ -110,7 +110,7 @@ ANIMAL_DELAY 0
 
 REVIVE_GRACE_PERIOD 3000
 
-
+FUN_ALLOWED 0

```

Comment:
By this not being here it's 0 by default (unless you specify a default value in the config datum). The whole config doesn't do disabled values like this for it to not be cluttered.